README.md

```diff
@@ -1,6 +1,6 @@
 # Cheddar
 
-![Cheddar logo](http://googledrive.com/host/0B3KE77--Zs7eSlN2MzF5ekdDZFE/)
```

Comment:
This fixes the Cheddar logo, it's currently just showing as a missing asset :) I don't see any reason why the adding of an image file would cause issues, but Java.... So just flagging this to check  😅

Reply:
That's fine as it is tiny. We generally avoid putting binaries into version-control if possible, because it inflates the size of data being pushed/pulled when interacting with the repo. ie: even if it gets deleted later, it's still in the repo history/diff stuff.